packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageList.tests.js

```diff
@@ -1205,9 +1205,9 @@ QUnit.module('MessageList', () => {
 
                         assert.roughEqual(scrollTop, scrollTopBefore, 1, 'scroll position should remain the same after updating typingUsers when not at the bottom');
                         done();
-                    });
-                });
-            });
+                    }, SCROLLVIEW_TIMEOUT);
```

Comment:
Module already has _resizeTimeout

Reply:
thanks for pointing out, i'll use it in cherry-pick and update in 25.1 in some future pr